scripts/systems/population_manager.gd

```diff
@@ -0,0 +1,76 @@
+extends Node
+
+const MIN_POPULATION = 3
+const MAX_POPULATION = 50
+const MIN_HUNGER = 0
+const MAX_HUNGER = 100
+const MIN_FULFILLMENT = 0
+const MAX_FULFILLMENT = 100
+
+@export_group("Starting values")
```

Comment:
starting values should be their own constants: for example we might want min/max hunger to be 0/100 but to start with 30 hunger.

Reply:
Once these are in a resource, these wont need to be export vars any more

Reply:
Also the init should probably assert that any starting value isnt less than the minimum or greater than the maximum.

Reply:
Done